data/json/proficiencies/wilderness.json

```diff
@@ -40,5 +40,43 @@
     "time_to_learn": "8 h",
     "default_time_multiplier": 2,
     "default_fail_multiplier": 1.2
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_bow_basic",
+    "name": { "str": "Basic Archer's Form" },
+    "description": "You have grasped some basic understanding on archery, and find handling bows easier.",
+    "can_learn": true,
+    "time_to_learn": "5 h",
+    "default_time_multiplier": 1.5,
+    "default_fail_multiplier": 1.2,
+    "category": "Archery",
+    "str_bonus": 1
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_bow_expert",
+    "name": { "str": "Expert Archer's Form" },
+    "description": "After significant practice, you have become an expert in archery.",
+    "can_learn": true,
+    "time_to_learn": "10 h",
+    "default_time_multiplier": 1.5,
+    "default_fail_multiplier": 1.2,
+    "required_proficiencies": [ "prof_bow_basic" ],
+    "category": "Archery",
+    "str_bonus": 1
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_bow_master",
+    "name": { "str": "Master Archer's Form" },
+    "description": "You are a master at the art of Archery.",
+    "can_learn": true,
+    "time_to_learn": "20 h",
+    "default_time_multiplier": 1.5,
+    "default_fail_multiplier": 1.2,
+    "required_proficiencies": [ "prof_bow_expert" ],
+    "category": "Archery",
+    "str_bonus": 1
```

Comment:
I think the syntax here might work better as a nested object: like,
```
"bonuses": {
  "category": "Archery",
  "str_bonus": 1
}
```
That makes the context of `category` clearer, and it leaves room for us later allowing an array there so that a proficiency might have multiple category bonuses with different associated things.

Reply:
Did you see my suggestion here? 
https://github.com/CleverRaven/Cataclysm-DDA/pull/50064#discussion_r672365484

Reply:
I did not. Your way is better. GMTA.